docs/content/docs/contributor-docs/testing.md

```diff
@@ -164,67 +196,20 @@ make kind-test SERVICE=$SERVICE
 ```
 
 This provisions a Kubernetes cluster using `kind`, builds a container image with
-the ACK service controller, and loads the container image into the `kind` cluster.
+the ACK service controller, and loads the container image into the `kind`
+cluster.
 
-It then installs the ACK service controller and related Kubernetes manifests into
-the `kind` cluster using `kustomize build | kubectl apply -f -`.
+It then installs the ACK service controller and related Kubernetes manifests
+into the `kind` cluster using `kustomize build | kubectl apply -f -`.
 
-Then, the above script runs a series of test scripts that call `kubectl`
-and the `aws` CLI tools to verify that custom resources of the type managed by
-the respective ACK service controller is created, updated and deleted
-appropriately (still TODO).
+First, it will attempt to install the Helm chart for the controller to ensure
+the default values are safe and that the controller stands up properly.
 
-Finally, it will run tests that create resources for the respective service
-and verify if the resource has successfully created. In our example case it
-should create an S3 bucket and then destroy it again, yielding something like
-the following (edited down to the relevant parts):
-
-```bash
```

Comment:
nit: rather than removing old command output, it will be good to replace it with the new sample output.